docs/getting_started_guide/eo_products_properties.rst

```diff
@@ -0,0 +1,77 @@
+All of the parameters are string based. If a parameter requires an specific format it will be defined on the list below.
+
+----------------------------
+
+- **uid**: Unique identifier for the resource.
+- **productType**: A string identifying the entry type (e.g., ER02_SAR_IM__0P, MER_RR__1P, SM_SLC__1S, GES_DISC_AIRH3STD_V005).
+- **doi**: Digital Object Identifier identifying the product.
+- **platform**: A string with the platform short name (e.g., Sentinel-1).
+- **platformSerialIdentifier**: A string with the platform serial identifier.
+- **instrument**: A string identifying the instrument (e.g., MERIS, AATSR, ASAR, HRVIR, SAR).
+- **sensorType**: A string identifying the sensor type. Suggested values include: OPTICAL, RADAR, ALTIMETRIC, ATMOSPHERIC, LIMB.
+- **compositeType**: Type of composite product expressed as the time period that the composite product covers (e.g., P10D for a 10-day composite).
+- **processingLevel**: A string identifying the processing level applied to the entry.
+- **orbitType**: A string identifying the platform orbit type (e.g., LEO, GEO).
+- **spectralRange**: A string identifying the sensor spectral range (e.g., INFRARED, NEAR-INFRARED, UV, VISIBLE).
+- **wavelengths**: A number, set, or interval requesting the sensor wavelengths in nanometers.
+- **hasSecurityConstraints**: A string indicating if the resource has any security constraints. Possible values: TRUE, FALSE.
+- **dissemination**: Information about the dissemination level of the resource.
+- **title**: A name given to the resource.
+- **topicCategory**: Main theme(s) of the dataset.
+- **keyword**: Commonly used word(s) or formalized word(s) or phrase(s) used to describe the subject.
+- **abstract**: A brief narrative summary of the resource.
+- **resolution**: The spatial resolution of the resource.
+- **organisationName**: Name of the organization responsible for the resource.
+- **organisationRole**: Role of the organization (e.g., resourceProvider, custodian, owner, user, distributor, originator, pointOfContact, principalInvestigator, processor, publisher, author).
+- **publicationDate**: Date when the resource was published.
+- **lineage**: Information about the lineage of the resource.
+- **useLimitation**: Constraints on the usage of the resource.
+- **accessConstraint**: Restrictions and legal prerequisites for accessing the resource.
+- **otherConstraint**: Other constraints related to the resource.
+- **classification**: Security classification of the resource (e.g., unclassified, restricted, confidential, secret, topSecret).
+- **language**: Language of the resource content.
+- **specification**: Reference to the specification to which the resource conforms.
+- **parentIdentifier**: A string identifying the parent of the entry in a hierarchy of resources.
+- **productionStatus**: A string identifying the status of the entry (e.g., ARCHIVED, ACQUIRED, CANCELLED).
```

Comment:
```suggestion
```
Should be removed, see https://eodag.readthedocs.io/en/latest/changelog.html#v1-5-1-2020-04-08

Please also remove remaining occurrences of `productionStatus` in the code